README.md

```diff
@@ -0,0 +1,68 @@
+| version          |               String                |       **Yes**        |                                         | Version of Splunk to install                                                                                                                                                                                                                                             |
+| build            |               String                |       **Yes**        |                                         | Build number of the version                                                                                                                                                                                                                                              |
+| service_user     |               String                |          No          | `'splunk'`                              | User to run the Splunk process                                                                                                                                                                                                                                           |
+| service_password |               String                |          No          |                                         | Shadow hash of the password for the user to run the Splunk process. The resource will attempt to install Splunk as a system user, but some platforms or configurations may require a password.                                                                           |
+| source_url       |               String                |          No          | `'http://download.splunk.com/products'` | Base url to pull Splunk packages from. Use this if you are mirroring the downloads for Splunk packages. The resource will append the version, os, and filename to the url like so: `{source_url}/splunk/releases/0.0.0/linux/splunk-0.0.0-a1b2c3d4e5f6-Linux-x86_64.tgz` |
+| accept_license   |            true or false            | **Yes (on Windows)** | `false`                                 | Whether or not you accept Splunk's license agreement on install. Currently only for the Windows Package, as it requires the license to be accepted at install time. On Splunk for Linux, it can be accepted on first run with `--accept-license`.                        |
+
+##### Action *:uninstall*
+Removes Splunk or Universal Forwarder and all its configuration.
+
+Properties: _None_
+
+---
+
+Contributing
+------------
+
+Check out the [Github Guide to Contributing](https://guides.github.com/activities/contributing-to-open-source/)
+for some basic tips on contributing to open source projects, and make sure to read our [Contributing Guidelines](CONTRIBUTING)
+before submitting an issue or pull request.
+
+License and Authors
+-------------------
+- Author:: Alec Sears (alec.sears@cerner.com)
```

Comment:
You're the only author so far :) we'll add more as more people contribute significantly to it.